geoportal/geomapfish_geoportal/static-ngeo/js/apps/sass/desktop_alt.scss

```diff
@@ -52,9 +55,9 @@ header {
 
 div .ngeo-displaywindow {
   $displaywindow-min-height: 15rem;
-  left: $app-margin + $left-panel-width + $displaywindow-min-height / 2;
+  left: calc($app-margin + $left-panel-width + $displaywindow-min-height / 2);
   right: initial;
-  top: $topbar-height + $app-margin + 3 * $map-tools-size;
+  top: calc($topbar-height + $app-margin + 3 * $map-tools-size);
```

Comment:
It should be changed [here](https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/apps/desktop_alt/sass/desktop_alt.scss#L66).